### DISCUSSION TITLE
Testing plugins without hardware or mobile app

### DISCUSSION THREAD

**Discussion by dennisrijsdijk** [elgatosf/elgato#11] [Help] [answered]
Hey all, I'm working on a Stream Deck plugin at the moment and I was wondering if it's possible to test a button without having hardware or the mobile app. I've already run through my trial of the mobile app and it's not possible for me to buy it at the moment. Is there a way to test buttons within the Stream Deck application itself?

**Comment by ChekTek** [comment, marked as answer]
Hi!

If you reach out to streamdeck.elgato@corsair.com, and let them know you are working on a plugin, but ran out of a Stream Deck mobile subscription time, we can help you out.

Best,
Zack
